config.yaml

```diff
@@ -19,6 +20,7 @@ server:
       address: 127.0.0.1
       ports:
         - 110
+      timeout: 600 # in seconds
```

Comment:
Other than set it here you should add a config default in `main.go`:
https://github.com/trapped/gomaild2/blob/master/main.go#L30
as well as other defaults for POP3